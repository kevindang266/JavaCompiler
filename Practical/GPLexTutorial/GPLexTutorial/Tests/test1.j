int foo2 = 0;

if (foo2 < 31)
{
    foo2 = foo2 + 42;
}