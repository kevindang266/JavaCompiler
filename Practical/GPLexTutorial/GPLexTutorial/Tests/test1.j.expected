INT
IDENT (foo2)
'='
NUMBER (0)
';'
IF
'('
IDENT (foo2)
'<'
NUMBER (31)
')'
'{'
IDENT (foo2)
'='
IDENT (foo2)
'+'
NUMBER (42)
';'
'}'
EOF
